tests/update/prepare_system_for_update_tests.pm

```diff
@@ -24,6 +24,8 @@ sub run() {
     assert_screen('xterm-started');
     assert_script_sudo "chown $testapi::username /dev/$testapi::serialdev";
     assert_script_sudo "echo \"download.use_deltarpm = false\" >> /etc/zypp/zypp.conf";
+    # Refresh repodata - we use zypper, as pkcon might not be there. But they share metadata anyway
+    assert_script_sudo "zypper refresh";
```

Comment:
ah - right.. I missed to check what guiupdates_is_applicable filters on:

sub guiupdates_is_applicable() {
    return get_var("DESKTOP") =~ /gnome|kde|xfce|lxde/ && !check_var("FLAVOR", "Rescue-CD");
}

so the minimalx cases which might not have PK there is already filtered... pkcon should indeed be sufficient then. Thanks for the hint.